viewer/viewer.js

```diff
@@ -3580,6 +3585,204 @@ app.post('/updateUser/:userId', checkToken, function(req, res) {
   });
 });
 
+app.post('/uploadFragment', checkWebEnabled, fragmentUpload, function(req, res) {
+  Db.getUser(req.user.userId, function(err, user) {
```

Comment:
What kind of functions do you mean? The ones that return errors instead of doing res.send?